mono/metadata/culture-info.h

```diff
@@ -53,34 +53,65 @@ typedef struct {
 } DateTimeFormatEntry;
 
 typedef struct {
-	const stridx_t currency_decimal_separator;
-	const stridx_t currency_group_separator;
-	const stridx_t number_decimal_separator;
-	const stridx_t number_group_separator;
-
-	const stridx_t currency_symbol;
-	const stridx_t percent_symbol;
-	const stridx_t nan_symbol;
-	const stridx_t per_mille_symbol;
-	const stridx_t negative_infinity_symbol;
-	const stridx_t positive_infinity_symbol;
-
-	const stridx_t negative_sign;
-	const stridx_t positive_sign;
-
-	const gint8 currency_negative_pattern;
-	const gint8 currency_positive_pattern;
-	const gint8 percent_negative_pattern;
-	const gint8 percent_positive_pattern;
-	const gint8 number_negative_pattern;
-
-	const gint8 currency_decimal_digits;
-	const gint8 number_decimal_digits;
-
-	const gint currency_group_sizes [GROUP_SIZE];
-	const gint number_group_sizes [GROUP_SIZE];	
+	// 12x ushort -- 6 ints
```

Comment:
This was already discussed. It's exhausting you are trying to submit it in another PR. Revert that  `const` change.

Reply:
const is very unusual and useless and potentially harmful here. I have almost never seen this pattern, and I have seen recent compiler bugs associated with it.